### DISCUSSION TITLE
Kibana server is not ready yet

### DISCUSSION THREAD

**Discussion by tsurara36** [wazuh/wazuh-kibana-app#5385] [Q&A] [answered]
Hi all,
I am new to wazuh, and after installing the virtual machine OVA when I tried to enter the wazuh server, it says that the kibana server is not ready yet. I've checked and basically elasticsearch is not loaded.
![image](https://user-images.githubusercontent.com/131188770/232854280-561474f7-ceab-45eb-a074-771103f765c2.png)

I've also expanded the RAM of the VM to 11GB (I saw that system resource was one of the problems in a separate chat) so I don't think that is the problem. My versions are:
kibana version: 7.6.2
elasticsearch version: 7.6.2

**Comment by jctello** [comment, marked as answer]
Hi @tsurara36, 
Under certain circumstances the OVA's Indexing service may fail to start, specially with older, less refined versions.  The official Wazuh OVA used Elasticsearch 7.6.2 in versions 3.12 & 3.11 which were released before May 2020.

If you have a specific need to use this version, then you may simply restart the Elasticsearch service with `systemctl restart elasticsearch` and it should work correctly.

I however strongly encourage you to use the most recent Wazuh OVA with version 4.4.1 which you can find here: https://documentation.wazuh.com/current/deployment-options/virtual-machine/virtual-machine.html 

The newest Wazuh OVA uses the Wazuh Indexer instead of Elasticsearch. The Wazuh Indexer is a fully open-source repackaging of the OpenSearch project. The most current OVA also includes the many improvements that Wazuh has continued to provide.

**Reply by tsurara36** [reply]
Hi @jctello,
Thanks so much for your help, I have installed the latest OVA version and it works.
